City-of-Helsinki/drupal-hdbt
# UHF-2975: Fix the sms service icon

How to test:

1. `composer require drupal/hdbt:dev-UHF-2975_sms_service_layout_fix`
2. `make drush-cr`
3. Go to any instance where you have imported services, errand services and service channels.
4. Go to any service that has any errand service that has any service channel. Modify the class of the service channel so that the class of the service channel modifier is "service-channel--sms".
5. Make sure that this results into service channel that has same layout as in the example attached here.

![Screenshot 2021-10-28 at 15 06 45](https://user-images.githubusercontent.com/2276077/139252287-27894696-d1a7-4f51-8151-a8329259f12e.png)
